### PR TITLE
Only show directories in places window #1760

### PR DESCRIPTION
**Pull request by fallingrock**
Only show directories in places window #1760

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Back In Time
 
 Version 1.4.4-dev (development of upcoming release)
+* Fix: Only show directories in places window (#1760)
```
Comment (buhtz) [inline CHANGES]:
```suggestion
```
The bug you do fix here was never released. So there is not need to mention it in the Changelog.

File: CHANGES [hunk 2]
```diff
@@ -921,4 +922,4 @@ Version 0.5.1 (2008-10-10)
 * changed some texts
 
 Version 0.5 (2008-10-03)
-* This is the first release.
+* This is the first release.
```
Comment (buhtz) [inline CHANGES]:
```suggestion
* This is the first release.

```

File: qt/app.py [hunk 3]
```diff
@@ -2049,4 +2049,4 @@ def debugTrace():
     cfg.PLUGIN_MANAGER.appExit()
     appInstance.exitApplication()
 
-    logger.closelog()  # must be last line (log until BiT "dies" ;-)
+    logger.closelog()  # must be last line (log until BiT "dies" ;-)
```
Comment (buhtz) [inline qt/app.py]:
```suggestion
    logger.closelog()  # must be last line (log until BiT "dies" ;-)

```

**Review by buhtz — CHANGES_REQUESTED** [review]
Hello David,
On behalf of the team, thank you for your contribution and taking
the time to improve Back In Time. We appreciate it.

Your solution is quit obvious. Haven't seen this my self. :D

I added suggestions about minor modifications. We will [merge it after a creative break](https://github.com/bit-team/backintime/blob/dev/CONTRIBUTING.md#what-happens-after-you-opened-a-pull-request-PR).

Best,
Christian

File: qt/app.py (hunk 3 above)
Comment (buhtz) [reply qt/app.py]: See https://pylint.pycqa.org/en/latest/user_guide/messages/convention/missing-final-newline.html

Comment (fallingrock) [reply qt/app.py]: Intellij is adding that.  I'll adjust my settings for this project.

**Comment by buhtz** [comment]
Thanks a lot.
I canceled the TravisCI builds on purpose because to save credits.

**Review by buhtz — APPROVED** [review]

**Pull request merged**